### ISSUE TITLE
Can't connect please try again

### ISSUE THREAD

**Issue by julio8a** [bubuntux/nordvpn#108]
I'm getting this error message and I'm unsure what I'm doing wrong:
`Whoops! We couldn't connect you to 'United_States'. Please try again. If the problem persists, contact our customer support.`

Here's my docker file:
```
docker run -ti --cap-add=NET_ADMIN --cap-add=SYS_MODULE --device /dev/net/tun --name vpn \
            -e USER=myemail@gmail.com -e PASS='my-password' \
            -e CONNECT=United_States -e TECHNOLOGY=NordLynx -d bubuntux/nordvpn
```

Login info is correct as I'm able to login on my phone.

**Comment by bubuntux** [comment]
Rollback to version `3.7.4` take a look at #103

**Comment by julio8a** [comment]
Thank you for the information. 

It connects now!

I don't mean to bother, but I do have a question about the `Routing access without the web proxy.`

I run this:
```
docker run -ti --cap-add=NET_ADMIN --cap-add=SYS_MODULE --device /dev/net/tun --name vpn \
           -e USER=myemail@gmail.com -e PASS='my-password' \
	   -p 8080:80 \
	   -e NETWORK=192.168.86.0/24 \
           -e CONNECT=United_States -e TECHNOLOGY=NordLynx -d bubuntux/nordvpn:3.7.4
```

I did check my host network address and I inputted the correct one.

When I run deluge with `--net` on `vpn` I can't access the GUI for deluge on port 8112, or 8080. 
Is my mistake within port issues or putting in the wrong address?

I know this is outside of the project, but thought I'd ask. 

Thank you.
